lib/govuk_publishing_components/presenters/breadcrumb_selector.rb

```diff
@@ -45,9 +57,8 @@ def parent_breadcrumbs
         }
       end
 
-      def output
+      def best_match_option
```

Comment:
Could this just be called `options` rather than `best match options`?

Reply:
There is already an instance method called `options` which contains the collection of options. The purpose of this method is to determine which of those options should be used (and with which set of navigation data) in the current context. Hence the name.